int64 centerX
int64 centerY

---

int64[2] center
